app/js/services/services.js

```diff
@@ -585,6 +586,7 @@ KMCServices.factory('PlayerService', ['$http', '$modal', '$log', '$q', 'apiServi
 					'uiConf:description': data.description ? data.description : ''
 				};
 				request['uiConf:confVars'] = JSON.stringify({versions: playersService.getPlayerAndPluginsVersionObj(data), langs: data.playerLangCodes});
+				request['uiConf:html5Url'] = data.html5Url;
```

Comment:
what html5Url holds for latest and beta?

Reply:
Nothing - Empty string